grpc.go

```diff
@@ -520,6 +520,12 @@ func (a *agentGRPC) updateContainerConfigNamespaces(config *configs.Config, ctr
 		config.Namespaces = append(config.Namespaces, newUTSNs)
 	}
 
+	// If container needs to be in the agent PID namespace, do not create a new
+	// PID namespace.
+	if ctr.agentPidNs {
+		return
```

Comment:
I think we should log this as it has potential security implications.

Reply:
Makes sense. Will add a log message here.